ee/modules/350-node-local-dns/templates/configmap.yaml

```diff
@@ -20,7 +20,7 @@ data:
       reload 2s
       loop
       bind {{ .Values.global.discovery.clusterDNSAddress }} 169.254.20.10
-      forward . {{ .Values.global.discovery.clusterDNSAddress }} {{ .Values.global.discovery.clusterDNSAddress }} {{ .Values.global.discovery.clusterDNSAddress }} {
+      forward . {{ .Values.global.discovery.clusterDNSAddress }} {
```

Comment:
What was the reason behind repeating the DNS address three times?

Reply:
Shrouded in mystery. Written eons ago by @distol. Perhaps, it was related to a superstitious belief about multiple retries.